p2p/net/swarm/swarm_listen.go

```diff
@@ -127,6 +127,9 @@ func (s *Swarm) AddListenAddr(a ma.Multiaddr) error {
 		for {
 			c, err := list.Accept()
 			if err != nil {
+				if err != transport.ErrListenerClosed {
+					log.Error("swarm listener accept error: ", err)
```

Comment:
Should this include which listener this is? For example:
```suggestion
					log.Errorf("swarm listener for %s accept error: %s", a, err)
```

Reply:
You're right. I misread this as `log.Errorf("swarm listener %s accept error: %s", list, err)`. That makes sense.